web/webhook.go

```diff
@@ -38,7 +38,7 @@ func incomingWebhook(c *Context, w http.ResponseWriter, r *http.Request) {
 		}
 	}()
 
-	if strings.Split(contentType, "; ")[0] == "application/x-www-form-urlencoded" {
+	if strings.Split(contentType, ";")[0] == "application/x-www-form-urlencoded" {
```

Comment:
This seems to address the core issue, but I wonder if we might extend this solution to use https://golang.org/pkg/mime/#ParseMediaType and handle potentially other edge cases (i.e. case?)

Reply:
Hum, good point. I'll add this, I checked for something like that but failed to find it :)